test/docs/BUILD_OS_IMAGE.md

```diff
@@ -1,46 +1,69 @@
+# Creating a test image
 This document explains how to create base OS images and run test runners on them.
+This guide is written from the perspective of a Linux user, but it should work on Windows as well.
 
-For macOS, the host machine must be macOS. All other platforms assume that the host is Linux.
+## Prerequisites
+You need to have [QEMU](https://www.qemu.org/) installed.
 
-# Configuring a user in the image
-
-`test-manager` assumes that a dedicated user named `test` (with password `test`) is configured in any guest system which it should control.
-Also, it is strongly recommended that a new image should have passwordless `sudo` set up and `sshd`  running on boot,
-since this will greatly simplify the bootstrapping of `test-runner` and all needed binary artifacts (MullvadVPN App, GUI tests ..).
-The legacy method of pre-building a test-runner image is detailed [further down in this document](#).
-
-# Creating a base Linux image
+# Linux
 
 These instructions use Debian, but the process is pretty much the same for any other distribution.
 
+## Creating a virtual machine
+
 On the host, start by creating a disk image and installing Debian on it:
 
 ```
-wget https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-11.5.0-amd64-netinst.iso
-mkdir -p os-images
-qemu-img create -f qcow2 ./os-images/debian.qcow2 5G
-qemu-system-x86_64 -cpu host -accel kvm -m 4096 -smp 2 -cdrom debian-11.5.0-amd64-netinst.iso -drive file=./os-images/debian.qcow2
+$ wget -O debian.iso https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-11.5.0-amd64-netinst.iso
+$ qemu-img create -f qcow2 ./debian.qcow2 5G
+$ qemu-system-x86_64 -cpu host -accel kvm -m 4096 -smp 2 -cdrom debian.iso -drive file=./debian.qcow2
 ```
 
-## Dependencies to install in the image
+## Installing Linux
 
-`xvfb` must be installed on the guest system. You will also need
-`wireguard-tools` and some additional libraries. They are likely already
-installed if gnome is installed.
+Follow the distribution's installation process. The only important detail is to set up the [user](#User) correctly.
 
-### Debian/Ubuntu
+### User
+
+`test-manager` assumes that a dedicated user named `test` with password `test` exists.
 
+### sudo
+
+The user should be able to execute `sudo` without a password.
+
+One way of accomplishing this is to add the `test` user to the `wheel` group
+```bash
+$ gpasswd -a test wheel
+```
+and edit `/etc/sudoers` to allow members of `wheel` to execute commands without a password
+```bash
+$ sudo visudo
```

Comment:
To keep code blocks consistent with the rest.

```suggestion
gpasswd -a test wheel
```
and edit `/etc/sudoers` to allow members of `wheel` to execute commands without a password
```bash
sudo visudo
```